webpack/assets/javascripts/react_app/components/MemoryAllocationInput/index.js

```diff
@@ -1,3 +1,41 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import { setMemoryValue } from './MemoryAllocationInputActions';
+
+import reducer from './MemoryAllocationInputReducer';
+
 import MemoryAllocationInput from './MemoryAllocationInput';
 
-export default MemoryAllocationInput;
+// export reducers
+export const reducers = { memoryAllocationInput: reducer };
+
+const ConnectedMemoryAllocationInput = props => {
+  const dispatch = useDispatch();
+  const setMemoryAction = value => dispatch(setMemoryValue(value));
+
+  const memoryValue = useSelector(state => state.memoryAllocationInput.value);
+
+  useEffect(() => {
+    setMemoryAction(props.defaultValue);
+  }, [props.defaultValue]);
```

Comment:
missing `setMemoryAction`

Reply:
I don't think I understand what do you mean it is missing? where?